conf/defaults.yaml

```diff
@@ -281,3 +281,16 @@ pacemaker.thread.timeout: 10
 pacemaker.childopts: "-Xmx1024m"
 pacemaker.auth.method: "NONE"
 pacemaker.kerberos.users: []
+
+storm.resource.isolation.plugin: "org.apache.storm.container.cgroup.CgroupManager"
+
+# Configs for CGroup support
+storm.cgroup.hierarchy.dir: "/cgroup/storm_resources"
+storm.cgroup.resources:
+    - cpu
+    - memory
```

Comment:
With few exceptions, I see string values are quoted. Not sure this would make a difference with these values, but maybe we should be consistent.